terraform/region/kms.tf

```diff
@@ -1,7 +1,79 @@
+data "aws_kms_key" "access_log_key" {
+  key_id = "alias/mrk_access_logs_lb_encryption_key-${terraform.workspace}"
+}
+
+data "aws_iam_policy_document" "s3_loadbalancer_kms" {
```

Comment:
Could you please explain why you need to enable root and key administrator permissions? Is it not possible to make a policy which only allows access from the LB (i.e. the third statement)? Or does the policy have wider application beyond the LB?